examples/api/csharp/rcs/Rcs-Api-Console-Tester/RcsConsoleApp/RcsCodeSamples.cs

```diff
@@ -147,13 +147,13 @@ public static async Task ChangeReinforcedSectionLayout(RcsApiClient client)
 		{
 			#region changereinforcedcsslayout
 			//Get the list of avaliable reinforced cross-sections in the project
-			List<ReinforcedCrossSectionModel> reinforcedCrossSections = await client.GetProjectReinforcedCrossSectionsAsync(CancellationToken.None);
+			List<RcsReinfCssModel> reinforcedCrossSections = await client.GetProjectReinforcedCrossSectionsAsync(CancellationToken.None);
 
 			//Find the reinforced cross-section which we want to update in the project.
-			ReinforcedCrossSectionModel reinforcedCrossSection = reinforcedCrossSections.Where(x => x.Name == "RCS2").First();
+			RcsReinfCssModel reinforcedCrossSection = reinforcedCrossSections.Where(x => x.Name == "RCS2").First();
 
 			//Define the import settings
-			ReinfCssImportSetting importSetting = new ReinfCssImportSetting();
+			RcsReinfCssImportSetting importSetting = new RcsReinfCssImportSetting();
```

Comment:
For me
**ReinfCssImport**
or 
**RcsImportSetting**

do not mix RCS and ReinfCss. (RCS is the name of our application and  it means ReinforcedCrossSection (== ReinfCss))